.circleci/config.yml

```diff
@@ -450,9 +450,9 @@ workflows:
           context:
             - optimism
       - docker-publish:
-          name: deployer-release
+          name: integration-tests-release
```

Comment:
Is the deployer getting published at all in this?

Reply:
Indeed it is still being published, via the existing github action. This change prevents a nightly deployer tag from being published. We don't use the full deployer image on nightly anyways.